urumanifest/github.py

```diff
@@ -374,6 +375,8 @@ def nuke_recurse(nuke: Path):
     database.current_sha = rev
     database.valid = True
 
+ArtifactInfo = namedtuple('ArtifactInfo', 'path zipinfo name')
```

Comment:
I'd prefer to see this as a typed named tuple: https://docs.python.org/3/library/typing.html#typing.NamedTuple